src/targets/gpu/target.cpp

```diff
@@ -143,6 +144,7 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
 #endif
         dead_code_elimination{},
         enable_pass(mlir_enabled(), fuse_mlir{&ctx}),
+        enable_pass(mlir_enabled(), standalone_mlir{&ctx}),
```

Comment:
Why's this a separate pass instead of being a rewrite within `fuse_mlir()` (which could be renamed to something like `mlir_offload()`)?

What do folks think here?

Reply:
It is a good question. @manupak and I was discussing this today - i.e., whether to add use one pass or two ones. My argument was that adding `standalone conv` to  `fuse_mlir` was ambiguous. @manupak suggested that we could rename `fuse_mlir` as you mentioned. 

By and large, yes. We can rename `fuse_mlir` to `mlir_offload` which sounds more generic.